domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/BehandlingMetrics.kt

```diff
@@ -2,15 +2,15 @@ package no.nav.su.se.bakover.domain.behandling
 
 interface BehandlingMetrics {
     fun behandlingsstatusChanged(old: Behandling.BehandlingsStatus, new: Behandling.BehandlingsStatus)
-    fun incrementUnderkjentCounter()
+    fun incrementUnderkjentCounter(handling: UnderkjentHandlinger)
     fun incrementInnvilgetCounter(handling: InnvilgetHandlinger)
     fun incrementAvslåttCounter(handling: AvslåttHandlinger)
     fun incrementTilAttesteringCounter(handling: TilAttesteringHandlinger)
 
     enum class InnvilgetHandlinger {
         PERSISTERT,
         JOURNALFØRT,
-        OPPGAVE,
+        LUKKET_OPPGAVE,
```

Comment:
Tydeliggjorde statusene for OPPGAVE. Som gjør at vi må oppdatere Grafana.

Reply:
Har vi fått noe feedback på om disse metrikkene gir noen verdi enda? Ser potensialet for at dette stadig "bygges ut" til en stor infrastruktur, noe jeg synes vi bør stoppe med så fort som mulig med mindre det gir oss verdi. Telling av metrikkene bidrar vel heller ikke i en positiv retning når det kommer til mengden støy i koden.